bundles/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/eiscp/EispEventNotifier.java

```diff
@@ -0,0 +1,100 @@
+package org.openhab.binding.onkyo.internal.eiscp;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.openhab.binding.onkyo.internal.OnkyoEventListener;
+import org.openhab.binding.onkyo.internal.OnkyoStatusUpdateEvent;
+
+public class EispEventNotifier {
```

Comment:
Please add JavaDoc comment block above this line, describing this class and your `@author` tag.